### PR TITLE
[Refactor] process test doesn't depend on kafka

### PR DESCRIPTION
**Pull request by mproch**
nussknacker-kafka-flink-util depends on process module for tests, 
while process tests no longer depend on kafka. This way we can test kafka-based components more easily

### CODE REVIEW

File: engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkTestMainSpec.scala [hunk 1]
```diff
@@ -309,7 +309,8 @@ class FlinkTestMainSpec extends FunSuite with Matchers with Inside with BeforeAn
     SinkForInts.data should have length 0
   }
 
-  test("be able to test process with signals") {
+  //TODO: remove it
+  ignore("be able to test process with signals") {
```
Comment (lciolecki) [inline engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkTestMainSpec.scala:313]: Are you sure that is okey? Why you comment this test?

File: engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMainSpec.scala [hunk 2]
```diff
@@ -68,9 +68,8 @@ class SimpleProcessConfigCreator extends EmptyProcessConfigCreator {
     "genericSourceWithCustomVariables" -> WithCategories(GenericSourceWithCustomVariables)
   )
 
-  override def signals(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[TestProcessSignalFactory]] = Map("sig1" ->
-    WithCategories(new TestProcessSignalFactory(KafkaConfig("", None, None), "")))
-
+  override def signals(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[ProcessSignalSender]] =
```
Comment (lciolecki) [inline engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMainSpec.scala:71]: Why right now it's empty?

**Review by lciolecki — APPROVED** [review]
LGTM, but look at my comments.

File: engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMainSpec.scala (hunk 2 above)
Comment (mproch) [reply engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMainSpec.scala:71]: Tests were moved elsewhere

File: engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkTestMainSpec.scala (hunk 1 above)
Comment (mproch) [reply engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/runner/FlinkTestMainSpec.scala]: :+1: I moved test to kafka module

**Pull request merged**